x/contracts/simulator/ffi/ffi.go

```diff
@@ -94,7 +94,7 @@ func CreateContract(db *C.Mutable, path *C.char) C.CreateContractResponse {
 		}
 	}
 
-	err = contractManager.SetContract(context.TODO(), contractID, contractBytes)
+	err = contractManager.SetContractBytes(context.TODO(), runtime.ContractID(contractID[:]), contractBytes)
```

Comment:
nit
```suggestion
        contractID := runtime.ContractID(contractID[:])
	err = contractManager.SetContractBytes(context.TODO(), contractID, contractBytes)
```
this is specifically easier to read when separated out on github. If there aren't any real change requests though, we can always change this later.